eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -13,7 +13,9 @@ jobs:
         displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
         inputs:
           version: "$(DotNetCoreSDKVersion)"
-      - script: "dotnet pack eng/service.proj -o $(Build.ArtifactStagingDirectory) -warnaserror /p:ServiceDirectory=${{parameters.ServiceDirectory}} /p:PublicSign=false ${{ parameters.VersioningProperties }}"
+      - script: >-
```

Comment:
I'm not familiar with `>-` what does that mean? Does it allow for line wrapping or something?

Reply:
Yes, it just allows for wrapping the line